scss/_root.scss

```diff
@@ -150,6 +150,10 @@
   --#{$prefix}icon-filter: none;
   --#{$prefix}select-indicator: #{$form-select-indicator};
   --#{$prefix}select-disabled-indicator: #{$form-select-disabled-indicator};
+  --#{$prefix}table-striped-bg-factor: #{$table-striped-bg-factor};
```

Comment:
We'll see that at the end of the process of migrating everything, but these variables will probably end up in __tables.scss_.